services/videoService.js

```diff
@@ -0,0 +1,60 @@
+const { videoDao, classesDao } = require("../models")
+
+const getVideo = async ( classId ) => {
+    const searchClass = await classesDao.getClass(classId);
```

Comment:
변수명은 명사형으로 사용해 주세요. 결과적으로 return 값이 class에 대한 객체이므로 아래와 같이 고치면 좋을 것 같습니다.
```suggestion
    const class = await classesDao.getClass(classId);
```